src/server/trpc/router/admin.ts

```diff
@@ -68,6 +68,34 @@ export const adminRouter = router({
       });
     }),
 
+  deleteLocation: protectedStaffProcedure
+    .input(
+	  z.object({
+		id: z.number(),
+	  }),
+	)
+	.mutation(async ({ input, ctx }) => {
+	  return ctx.prisma.location.delete({
+		where: {
+		  id: input.id,
+		},
+	  });

```

Comment:
This is equivalent to writing raw SQL to delete from the database but no one wants to do that so we use `prisma` which allows us to just write typescript to interact with the database.